app/[locale]/members/page.tsx

```diff
@@ -33,14 +33,18 @@ const Magnifier: React.FC<SVGProps<SVGSVGElement>> = props => {
 
 const WelcomeMessage = () => {
   const t = useTranslations('members');
+  const localLang = useLocale();
   return (
-    <div className="flex flex-col justify-center bg-purple-100 dark:bg-gray-800 mt-16 mb-16 h-20">
-      <div className="flex justify-evenly flex-wrap">
-        <p className="text-2xl">{t('paragraph')}</p>
+    <div
+      dir={localLang === 'he' ? 'rtl' : 'ltr'}
+      className="flex flex-col justify-center bg-purple-100 dark:bg-gray-800 mb-6 mt-2 md:mb-12 py-8 px-4 md:p-4"
```

Comment:
Can you extract this into some utility function? Seems like we will use this part a lot

Reply:
I think it's better to use it locally where it's needed in the file. Just adding one import of the useLocale hook.
But if you think you can improve by extracting this logic into a separated file give it a go.